app_server/app/Router.js

```diff
@@ -51,6 +51,25 @@ server.register({
   }
 });
 
+/* Register static file handler */
+server.register(require('inert'), function(err) {
+  if (err) {
+    logger.error('Unable to register inert file handler');
+  }
+
+  server.route({
+    method: 'GET',
```

Comment:
Might be a better idea to throw an error and crash loudly if the plugin failed to be registered. And the `server.route` called can be moved outside or at least shouldn't be reached if there is error.